examples/arduino/sketches/sketch_obstacleAvoidance.ino

```diff
@@ -0,0 +1,25 @@
+#include <Smartcar.h>
+#include <VL53L0X.h>
+#include <Wire.h>
+
+constexpr float cruise_speed = 40;
+constexpr unsigned trigger_dist = 300;
+
+BrushedMotor leftMotor{smartcarlib::pins::v2::leftMotorPins};
+BrushedMotor rightMotor{smartcarlib::pins::v2::rightMotorPins};
+DifferentialControl control{leftMotor, rightMotor};
+SimpleCar car{control};
+VL53L0X sensor;
+
+void setup() {
+Serial.begin(9600);
+Wire.begin();
+while(!sensor.init());
```

Comment:
Might look odd to some people to have an empty statement following a `while`
Won't complain tho; it's in my style for that kind of situations